packages/headless/src/features/generated-answer/generated-answer-actions.ts

```diff
@@ -125,6 +128,16 @@ export const updateResponseFormat = createAction(
     })
 );
 
+const nonEmptyArray = new ArrayValue({
```

Comment:
Does this really require a non-empty array or just an array of non-empty strings?
Does it reject `[]`?

Reply:
Also it seems duplicated now since you also added it in utils

Reply:
Indeed this was duplicated, fixed it now!

For your first question, this just require an array of non-empty strings, so [] will not be rejected.